src/evo/neuron.py

```diff
@@ -325,10 +325,14 @@ def neuron_finder(cortical_area, location, radius):
     for key in runtime_data.brain[cortical_area]:
         x = brain[cortical_area][key]['location'][0]
         y = brain[cortical_area][key]['location'][1]
-        # z = brain[cortical_area][key]['location'][2]
+        z = brain[cortical_area][key]['location'][2]
 
         # Searching only the XY plane for candidate neurons         ????
-        if np.sqrt((x - location[0]) ** 2 + (y - location[1]) ** 2) <= (radius ** 2):
+        if np.sqrt(
+            (x - location[0]) ** 2 + \
+            (y - location[1]) ** 2 + \
+            (z - location[2]) ** 2
+        ) <= (radius ** 2):
             if collections.Counter(neuron_list)[key] == 0:
                 neuron_list = np.append(neuron_list, key)
 
```

Comment:
I was initially using this `neuron_finder` function in early drafts of the distance processor - that's the only reason I changed this code.